sklearn/gaussian_process/tests/test_gaussian_process.py

```diff
@@ -129,7 +129,7 @@ def test_ordinary_kriging():
 
 def test_no_normalize():
     gp = GaussianProcess(normalize=False).fit(X, y)
```

Comment:
@zhengruifeng your fix looks fine. Can you follow @jnothman suggestion about the test?

If I were you I would leave `test_no_normalize` unmodified and just add a test function like this:

```py
def test_batch_size():                                                                                                                
    # TypeError when using batch_size on Python 3, see                                                                                
    # https://github.com/scikit-learn/scikit-learn/issues/7329 for more            
    # details                                                                                                                         
    gp = GaussianProcess()                                                                                                            
    gp.fit(X, y)                                                                                                                      
                                                                                                                                      
    gp.predict(X, batch_size=1)                                                                                                       
                                                                                                                                      
    gp.predict(X, batch_size=1, eval_MSE=True) 
```